charts/tpu-dra-driver/README.md

```diff
@@ -0,0 +1,8 @@
+# TPU DRA DRIVER
+
+This helm chart is for running TPU DRA Driver Private Preview version on GKE
+
+## Overview
```

Comment:
mention version restriction for GKE to be 1.32+. Also mention that the production device plugin needs to be disabled before installing this one.